segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/stat/AutoScalerConfig.java

```diff
@@ -30,6 +30,7 @@
     public static final Property<String> TLS_CERT_FILE = Property.named("tlsCertFile", "");
     public static final Property<Boolean> AUTH_ENABLED = Property.named("authEnabled", false);
     public static final Property<String> TOKEN_SIGNING_KEY = Property.named("tokenSigningKey", "secret");
+    public static final Property<Boolean> VALIDATE_HOSTNAME = Property.named("validateHostName", false);
```

Comment:
By the principle of secure-by-default, I would suggest that validation be enabled by default.

Reply:
I have brought this up in earlier discussions as well. Here I go again :) - very few products in our ecosystem use (including Pravega) are secure by default.

Reply:
> We are not on the same page here. I'm -1 on not being secure by default.

The default for `authEnabled` and `tlsEnabled` is `false`. Do you think that should be `true`? If yes, I will create another issue to fix this.

Reply:
My suggestion is simply that `validateHostName` be enabled by default, but that standalone's `tlsEnabled` be disabled by default.